HW_1/task_2.py

```diff
@@ -0,0 +1,25 @@
+#Напишите код, который запрашивает число и сообщает является ли оно простым или составным.
+#Используйте правило для проверки: “Число является простым, если делится нацело только на единицу
+#и на себя”. Сделайте ограничение на ввод отрицательных чисел и чисел больше 100 тысяч.
+n = int(input('Введите число от 0 до 100000: '))
+count = 0
+
+
+if n == 0 or n == 1:
+    print('Число ', n,'не является ни простым, ни составным')
+
+
+if n >= 0 and n <= 100000:
+    for i in range(1,n+1):
```

Comment:
Много итераций. Вспомните теорию чисел
```suggestion
    for i in range(1, n + 1):
```